src/utils/transitionMappings.js

```diff
@@ -0,0 +1,36 @@
+const lu = {
```

Comment:
Where is the information for this file coming from? How about putting this into CSV files in `/lib/data` and provide a lib function to make this information also available outside the app.

Reply:
I I moved the mappings to csv files in the profile folders in tree-lib and write them into types